docker-for-mac/apple-m1.md

```diff
@@ -1,16 +1,64 @@
 ---
-description: Apple M1 Tech Preview
+description: Apple M1
 keywords: Docker Desktop, M1, Silicon, Apple, tech preview, dev preview
-title: Apple M1 Tech Preview
+title: Apple M1
 toc_min: 2
 toc_max: 3
 ---
 
-Welcome to the tech preview of Docker Desktop for Apple M1. This tech preview is aimed at early adopters of Apple M1 machines, who would like to try an experimental build of Docker Desktop.
+Welcome to Docker Desktop for Apple Silicon.
```

Comment:
Can we use M1 and Silicon interchangeably? The title has M1, but the description refers to Silicon.

Reply:
I was thinking in an ideal world we would use "Apple Silicon" because "M1" is their first Apple Silicon chip, and there will be more soon. I've seen references online to an "M1X".